### PR TITLE
[Agent] Change monitoring defaults

### PR DESCRIPTION
**Pull request by ruflin**
WIP

### CODE REVIEW

File: x-pack/elastic-agent/pkg/core/plugin/app/monitoring/beats/beats_monitor.go [hunk 1]
```diff
@@ -35,7 +41,7 @@ func NewMonitor(downloadConfig *artifact.Config) *Monitor {
 	return &Monitor{
 		operatingSystem: downloadConfig.OS(),
 		installPath:     downloadConfig.InstallPath,
-		config:          &monitoringConfig.MonitoringConfig{},
+		config:          &defaultMonitoringConfig,
```
Comment (ruflin) [inline x-pack/elastic-agent/pkg/core/plugin/app/monitoring/beats/beats_monitor.go:44]: As this introduces defaults, I wonder if we still need it to be a reference on line 35?

**Comment by ph** [comment]
@ruflin Is this PR still valid?

**Comment by ruflin** [comment]
@ph I think yes, as we still rely on the defaults in config files instead of in code. @michalpristas @blakerouse Could one of you take this over?

**Comment by michalpristas** [comment]
i will take a look but we are in most cases using baked-in defaults to allow empty configs (this is a recent change) will align it with proposal here

**Comment by michalpristas** [comment]
Created new PR here: https://github.com/elastic/beats/pull/18927

**Pull request closed without merging**